vms/event/templates/event/list.html

```diff
@@ -25,7 +25,9 @@
                         <td>{{ event.name }}</td>
                         <td>{{ event.start_date }}</td>
                         <td>{{ event.end_date }}</td>
-						<td>{{ event.city }}</td>
+		        <td> 

```

Comment:
At line 17, put the appropriate heading.
Another thing is display city in that column and on the event-name, we can have a hyperlink thal will take the user to this new page. What do you say @anjali-dhanuka @ExactlyMay

Reply:
@abhishekarya286 In VMS, we are using buttons everywhere to show details, hyperlinks can be found nowhere. Don't you think it will look inconsistent.